src/interpolations.jl

```diff
@@ -1852,10 +1874,84 @@ edgedof_interior_indices(::RaviartThomas{RefTriangle, 2}) = ((1, 2), (3, 4), (5,
 facedof_interior_indices(::RaviartThomas{RefTriangle, 2}) = ((7, 8),)
 adjust_dofs_during_distribution(::RaviartThomas{RefTriangle, 2}) = true
 
-function get_direction(::RaviartThomas{RefTriangle, 2}, j, cell)
-    j > 6 && return 1
-    edge = edges(cell)[(j + 1) ÷ 2]
-    return ifelse(edge[2] > edge[1], 1, -1)
+function get_direction(::RaviartThomas{RefTriangle, 2}, shape_nr, cell)
+    shape_nr > 6 && return 1
+    edge_nr = (shape_nr + 1) ÷ 2
+    return get_edge_direction(cell, edge_nr)
+end
+
+# RefQuadrilateral, 1st order Lagrange
+edgedof_indices(ip::RaviartThomas{RefQuadrilateral}) = edgedof_interior_indices(ip)
+facedof_indices(ip::RaviartThomas{RefQuadrilateral}) = (ntuple(i -> i, getnbasefunctions(ip)),)
+
+# https://defelement.org/elements/examples/quadrilateral-raviart-thomas-lagrange-1.html
+function reference_shape_value(ip::RaviartThomas{RefQuadrilateral, 1}, ξ::Vec{2, T}, i::Int) where {T}
+    x, y = ξ
+    nil = zero(T)
+
+    i == 1 && return Vec(nil, -(1 - y) / 4) # Changed sign, follow Ferrite's sign convention
+    i == 2 && return Vec((1 + x) / 4, nil)  # Changed sign, follow Ferrite's sign convention
+    i == 3 && return Vec(nil, (1 + y) / 4)
+    i == 4 && return Vec((x - 1) / 4, nil)
+    throw(ArgumentError("no shape function $i for interpolation $ip"))
+end
+
+getnbasefunctions(::RaviartThomas{RefQuadrilateral, 1}) = 4
+edgedof_interior_indices(::RaviartThomas{RefQuadrilateral, 1}) = ((1,), (2,), (3,), (4,))
+facedof_interior_indices(::RaviartThomas{RefQuadrilateral, 1}) = ((),)
+adjust_dofs_during_distribution(::RaviartThomas{RefQuadrilateral, 1}) = false
+
+function get_direction(::RaviartThomas{RefQuadrilateral, 1}, shape_nr, cell)
+    return get_edge_direction(cell, shape_nr) # shape_nr = edge_nr
+end
+
+# RefTetrahedron, 1st order Lagrange
+# https://defelement.com/elements/examples/tetrahedron-raviart-thomas-lagrange-1.html
+function reference_shape_value(ip::RaviartThomas{RefTetrahedron, 1}, ξ::Vec{3}, i::Int)
+    x, y, z = ξ
+    i == 1 && return Vec(2 * x, 2 * y, 2 * (z - 1)) # Changed sign, follow Ferrite's sign convention
+    i == 2 && return Vec(2 * x, 2 * (y - 1), 2 * z)
+    i == 3 && return Vec(2 * x, 2 * y, 2 * z)
+    i == 4 && return Vec(2 * (x - 1), 2 * y, 2 * z) # Changed sign, follow Ferrite's sign convention
+    throw(ArgumentError("no shape function $i for interpolation $ip"))
+end
+
+getnbasefunctions(::RaviartThomas{RefTetrahedron, 1}) = 4
+edgedof_interior_indices(::RaviartThomas{RefTetrahedron, 1}) = ntuple(_ -> (), 6)
+edgedof_indices(ip::RaviartThomas{RefTetrahedron, 1}) = edgedof_interior_indices(ip)
+facedof_interior_indices(::RaviartThomas{RefTetrahedron, 1}) = ((1), (2), (3), (4))
```

Comment:
This should have been caught by the tests, will fix in separate pr. 
```suggestion
facedof_interior_indices(::RaviartThomas{RefTetrahedron, 1}) = ((1,), (2,), (3,), (4,))
```